### DISCUSSION TITLE
Help with generating datasets with care gaps

### DISCUSSION THREAD

**Discussion by amaankazzii** [synthetichealth/synthea#1431] [General]
Hi! I am working on generating healthcare data that I want to output in an FHIR format. I have been trying to generate data consisting of care gaps (missed screenings, missed vaccines, etc) but for some reason data doesn't contain any care gaps.

Any help or direction in what command line argument I should pass would be greatly appreciated. Thank you!

**Comment by jawalonoski** [comment]
Synthea doesn't simulate or record missed, canceled, or rescheduled appointments.

**Reply by annayudovin** [reply]
Next question: I assume, from reading [#1111](https://github.com/synthetichealth/synthea/discussions/1111), that [encounterModule.java](https://github.com/synthetichealth/synthea/blob/d4af94570aba5ba9b199d0071b6ca8be0763e441/src/main/java/org/mitre/synthea/modules/EncounterModule.java) would be the place to try to introducing "missed" care. While I can already see that trying to model  appointments that didn't actually take place would be problematic, introducing care gaps of a few years might be easily doable.

However, would I be correct in assuming that the usefulness of doing this would be limited by the same factor as the "Loss of Care" feature? Essentially, that what happens during a care gap would have to be defined on a by-condition basis, and that without these detailed models of condition progression in the absence of care, the resulting data wouldn't actually be simulating anything interesting about the real world?

**Reply by dehall** [reply]
Yes you're correct. In terms of just "the data is missing" you can modify various places such as the EncounterModule to make that happen, but what happens after that almost certainly won't be realistic
